packages/connect-session/src/utils/base64.ts

```diff
@@ -0,0 +1,58 @@
+import base64 from 'base64-js'
+
+export function padding(str) {
```

Comment:
If we are going to use these utils, will need tests, general code tidy up and type notations - this package is just too mission critical without